src/Traits/Helpers.php

```diff
@@ -78,14 +78,28 @@ private function getInstallMessage($npmInstall)
     private function installDependencies($npmInstall)
     {
         $depsInstalled = $this->dependenciesInstalled();
-
-        if ($npmInstall || (!$npmInstall && !$depsInstalled)) {
-            $this->runProcessInBlast([
-                'npm',
-                'ci',
-                '--production',
-                '--ignore-scripts',
-            ]);
+        if (preg_match('/home\/vagrant/', getcwd())) {
```

Comment:
Hi @RobertByrnes, this is cool! I think this kind of hint can definitely save some debugging time. Although, I'm wondering if it isn't too Windows-specific? Would it make sense to wrap `$this->runProcessInBlast()` in a try/catch and provide the hint only if the command fails?

Reply:
@pboivin Good call, this way platform specific messages could be displayed should they be needed. Will make edit this evening